src/Pagination.xml

```diff
@@ -0,0 +1,65 @@
+            <description>Hide paging buttons when there is only one page</description>
+        </property>
+        <property key="pagingStyle" type="enumeration" defaultValue="default">
+            <caption>Paging style</caption>
+            <category>Appearance</category>
+            <description>Default rendering as grid</description>
+            <enumerationValues>
+                <enumerationValue key="default">Default</enumerationValue>
+                <enumerationValue key="custom">Custom</enumerationValue>
+            </enumerationValues>
+        </property>
+        <property key="items" type="object" isList="true" required="false">
+            <caption>Items</caption>
+            <category>Custom</category>
+            <description/>
+            <properties>
+                <property key="item" type="enumeration" defaultValue="firstButton">
+                    <caption>Item</caption>
+                    <category>General</category>
+                    <description>Item of the pagination</description>
+                    <enumerationValues>
+                        <enumerationValue key="firstButton">First button</enumerationValue>
+                        <enumerationValue key="lastButton">Last button</enumerationValue>
+                        <enumerationValue key="nextButton">Next button</enumerationValue>
+                        <enumerationValue key="previousButton">Previous button</enumerationValue>
+                        <enumerationValue key="text">Text</enumerationValue>
+                        <enumerationValue key="pageNumberButtons">Page number buttons</enumerationValue>
+                    </enumerationValues>
+                </property>
+                <property key="text" type="translatableString" required="false">
+                    <caption>Text</caption>
+                    <category>General</category>
+                    <description>Use placeholder: {firstItem} {lastItem} {totalItems} {currentPageNumber} {totalPages}</description>
+                    <translations>
+                        <translation lang="en_US">{firstItem} to {lastItem} of {totalItems}</translation>
+                        <translation lang="nl_NL">{firstItem} tot {lastItem} van {totalItems}</translation>
+                    </translations>
+                </property>
+                <property key="showIcon" type="enumeration" defaultValue="default">
+                    <caption>Show icon</caption>
+                    <category>General</category>
+                    <description></description>
+                    <enumerationValues>
+                        <enumerationValue key="default">Default</enumerationValue>
+                        <enumerationValue key="none">None</enumerationValue>
+                    </enumerationValues>
+                </property>
+            </properties>
+        </property>
+        <property key="maxPageButtons" type="integer" defaultValue="7">
```

Comment:
Should be one level up, as it part of the items. Add conditional visibility in WM only show for `item === "pageNumberButtons"`

Reply:
Done